content/en/platform/corda/5.0/deploying-operating/topology/_index.md

```diff
@@ -9,4 +9,71 @@ menu:
+
+Performance testing of your CorDapps under expected loads is required to determine the values that you will require in a production deployment.
+
+
+{{<
+  figure
+  src="aws.png"
+  width=100%
+  alt="AWS Example"
+  figcaption="AWS Example"
+>}}
+
+
+
+{{<
+  figure
+  src="azure.png"
+  width=100%
+  alt="Azure Example"
+  figcaption="Azure Example"
+>}}
+
+
+1.  External Network:
+    * Represents either the Internet, or a known peered network.
+2.  Load Balancer Firewall:
+    * Represents the network rules used to limit which networks are allowed to access the load balancer.
+    * AWS resource provided by "EC2/Security Groups".
+    * Azure resource provided by "Load balancing/Load Balancers", or "Network security groups".
+    * Create your own, or use ones created at stage 5.
+3.  Load Balancer:
+    * Represents a load balancer that manages ingress traffic.
+    * AWS resource provided by "EC2/Load Balancers".
+    * Azure resource provided by "Load balancing/Load Balancers".
+    * Create your own, or use one created at stage 5.
+4.  Application Ingress Firewall:
+    * Represents the network rules used to limit which networks are allowed to access the application network ingress.
+    * AWS resource provided by "EC2/Security Groups".
+    * Azure resource provided by "Network security groups".
+    * Create your own, or use ones created at stage 5.
+5.  Application Ingress:
+    * Represents the Kubernetes Ingress Controller, or Kubernetes Service that exposes the cluster applications to the outside network.
+    * Usually accompanied with a cloud-native load balancer with the ability to configure further.
+    * Kubernetes resources provided by, but not limited to, "Ingress-Nginx", "Traefik Proxy", and the Kubernetes resource kind "Service" (type: LoadBalancer).
+6.  Corda 5:
+    * TBC
+7.  Database Firewall:
+    * Represents the network rules used to limit which networks are allowed to access the database.
+    * AWS resource provided by "EC2/Security Groups".
+    * Azure resource provided by "Network security groups".
+8.  Database:
+    * Represents the datastore used by Corda 5.
+    * Supported database engine is "PostgreSQL" (version 14.4).
+    * AWS resource provided by "RDS".
+    * Azure resource provided by "Azure Database for PostgreSQL flexible servers".
+9.  Egress Firewall (optional limitation):
+    * Represents the network rules used to limit egress network access to external networks.
+    * AWS resource provided by "EC2/Security Groups".
+    * Azure resource provided by "Load balancing/Load Balancers", or "Network security groups".
+10. Egress:
+    * Represents the route external egress network traffic takes from the application network.
+    * In AWS it is routed (from Subnet: B) via a NAT gateway, and, if the destination is on the Internet, an Internet gateway.
```

Comment:
> * In AWS it is routed (from Subnet: B) via a NAT gateway, and, if the destination is on the Internet, an Internet gateway.

"Subnet: B" was updated to "Subnet: Private" in the diagrams after review from @JasonSinclair95